mediaflow_proxy/extractors/vidoza.py

```
@@ -0,0 +1,42 @@
+from urllib.parse import urlparse
+
+from mediaflow_proxy.extractors.base import BaseExtractor, ExtractorError
+
+
+class VidozaExtractor(BaseExtractor):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        # Use segment_endpoint since final URL is a direct .mp4
+        self.mediaflow_endpoint = "stream"
+
+    async def extract(self, url: str, **kwargs) -> Dict[str, Any]:
```

Ruff (0.14.5)

Unused method argument: `kwargs`

(ARG002)